Tasks/FtpUpload/package.json

```diff
@@ -15,6 +15,6 @@
   "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
   "dependencies": {
     "vsts-task-lib": "^0.9.20",
-    "ftp": "*"
+    "ftp": "0.3.10"
```

Comment:
are we sure we want to lock into this version of the ftp module? would "^0.3.10" or even "~0.3.10" be better?

Reply:
Good question.  We've had many bugs introduced with vsts-task-lib having an unexpected behavioral change.  It's probably best to consider this on a case-by-case basis.  FWIW, my recommendation with vsts-task-lib is to pin to a specific version (maybe I should make that adjustment with this change).  I'm not sure it matters either way with this particular module (it hasn't been updated for 3 years).